test/test-lib-httpd.sh

```diff
@@ -21,7 +21,7 @@ function test_httpd_integration() {
 # Check the imagestream
 function test_httpd_imagestream() {
   case ${OS} in
-    rhel7|centos7) ;;
+    rhel7|centos7|rhel8) ;;
```

Comment:
And what about rhel9?

Reply:
I would say not now for now.

Reply:
@zmiklank I would like to test general imagestreams only for RHEL8 and later on add RHEL9. Step by step.